shell/artifacts/Products/ExperimentalProducts.recipes

```diff
@@ -0,0 +1,41 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+// import 'Arrivinator.manifest'
```

Comment:
Thanks for splitting them into a separate file, i really like this.
maybe since it is called "experimental" it is ok to uncomment the recipes?